include/enums.h

```diff
@@ -2921,38 +2921,20 @@ enum EncounterStatusFlags {
 };
 
 enum WindowFlags {
-    WINDOW_FLAGS_1                 = 0x00000001,
-    WINDOW_FLAGS_2                 = 0x00000002,
-    WINDOW_FLAGS_4                 = 0x00000004,
-    WINDOW_FLAGS_8                 = 0x00000008,
-    WINDOW_FLAGS_10                = 0x00000010,
-    WINDOW_FLAGS_20                = 0x00000020,
+    WINDOW_FLAGS_INITIALIZED       = 0x00000001,
+    WINDOW_FLAGS_FPUPDATE_CHANGED  = 0x00000002,
+    WINDOW_FLAGS_HIDDEN            = 0x00000004, // updated but not rendered
+    WINDOW_FLAGS_INITIAL_ANIMATION = 0x00000008,
+    WINDOW_FLAGS_HAS_CHILDREN      = 0x00000010,
+    WINDOW_FLAGS_DISABLED          = 0x00000020, // not updated or rendered
```

Comment:
These comments can be written as doxygen comments:
```suggestion
    WINDOW_FLAGS_DISABLED          = 0x00000020, ///< Not updated or rendered
```